README.md

```diff
@@ -12,4 +12,14 @@ The testing repo for Seine Bot
 
 Demos below have more details:
 
-add PR request review should automatically run when you make a PR 
+*add PR request review should automatically run when you make a PR*
+
+after you push your commits. on github (or remotely) create a new PR with your commited changes. Once you create the PR, the bot should automatically run.
```

Comment:
@SeineSailor any improvements on grammar and tone?

debug notes: I moved the secrets to job level in llm-review.yml.